### PR TITLE
Selenium: Unexpected fail of TransitiveDependencyTest in the selenium tests

### PR DESCRIPTION
**Pull request by Ohrimenko1988**
### What does this PR do?
Unexpected fail of TransitiveDependencyTest in the selenium tests

### What issues does this PR fix or reference?
Issue: https://github.com/eclipse/che/issues/9258

### CODE REVIEW

**Review by SkorikSergey — APPROVED** [review]

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/projectexplorer/dependencies/TransitiveDependencyTest.java [hunk 1]
```diff
@@ -92,10 +99,17 @@ public void transitiveDependencyTest() throws Exception {
...
-    loader.waitOnClosed();
-    for (int i = 36; i <= 40; i++) {
-      editor.setCursorToLine(i);
-      editor.selectLineAndDelete();
-    }
+    Actions action = actionsFactory.createAction(seleniumWebDriver);
+    editor.setCursorToLine(36);
+    action
+        .keyDown(SHIFT)
+        .sendKeys(DOWN)
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/projectexplorer/dependencies/TransitiveDependencyTest.java]: How about replacing multiple `sendKeys(DOWN)` invocations by the one command `sendKeys(DOWN, DOWN, DOWN, DOWN, DOWN)` ?

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/projectexplorer/dependencies/TransitiveDependencyTest.java [hunk 2]
```diff
@@ -92,10 +99,17 @@ public void transitiveDependencyTest() throws Exception {
 
   private void deleteDependency() {
     editor.waitActive();
-    loader.waitOnClosed();
-    for (int i = 36; i <= 40; i++) {
-      editor.setCursorToLine(i);
-      editor.selectLineAndDelete();
-    }
+    Actions action = actionsFactory.createAction(seleniumWebDriver);
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/projectexplorer/dependencies/TransitiveDependencyTest.java]: This variable `action` looks redundant so as it is used just ones.

Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/projectexplorer/dependencies/TransitiveDependencyTest.java]: What about moving `actionsFactory.createAction(seleniumWebDriver)` to the **SeleniumWebDriverHelper** class?

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/projectexplorer/dependencies/TransitiveDependencyTest.java [hunk 3]
```diff
@@ -50,6 +55,8 @@
   @Inject private Loader loader;
   @Inject private PopupDialogsBrowser popupDialogsBrowser;
   @Inject private TestProjectServiceClient testProjectServiceClient;
+  @Inject private ActionsFactory actionsFactory;
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/projectexplorer/dependencies/TransitiveDependencyTest.java]: Are we really need this field after moving create action code to the SeleniumwebDriverHelper?

**Review by dmytro-ndp — APPROVED** [review]

**Pull request merged**